### DISCUSSION TITLE
Add snapping of items when being trimmed / resized close to another item

### DISCUSSION THREAD

**Discussion by Ahmed-Ateeq** [samuelarbibe/dnd-timeline#62] [Ideas] [answered]
Hello,

I was wondering that do we currently have support for snapping while the items are being trimmed closer to each other.

**Comment by khanzzirfan** [comment, marked as answer]
@Ahmed-Ateeq , does this fit to your idea? Grid size has support for snapping elements.
https://samuel-arbibe.gitbook.io/dnd-timeline/documentation/timelinecontext#rangegridsizedefinition
